app/views/xur.html

```diff
@@ -4,10 +4,10 @@
     <div ng-repeat="category in vm.categoryOrder">
       <h2 ng-bind="category"></h2>
       <div class="items">
-        <div class="xurItem" ng-repeat="saleItem in vm.itemCategories[category] track by saleItem.item.index">
+        <div class="xurItem" align="right" ng-repeat="saleItem in vm.itemCategories[category] track by saleItem.item.index">
           <dim-simple-item id="xur-{{::saleItem.item.index}}" item-data="saleItem.item" ng-click="vm.itemClicked(saleItem.item, $event)"></dim-simple-item>
           <div class="cost" ng-class="{notenough: (vm.totalCoins[saleItem.currency.itemHash] < saleItem.cost)}">
-            {{::saleItem.cost}}/{{vm.totalCoins[saleItem.currency.itemHash]}}
+            {{::saleItem.cost}}
```

Comment:
Either something like this:
![screenshot 2016-10-04 21 20 50](https://cloud.githubusercontent.com/assets/4798491/19104945/b009b9dc-8a9c-11e6-90bc-f90833c63474.png)

or a footer on the bottom of it showing the same info.
It is redundant to have it show 13/90 on every item.